k0765065/assignment02.py

```diff
@@ -1,10 +1,22 @@
-statement = "You are"
-days_in_a_year = 365
-hours_in_a_day = 24
-days_of_this_year = 231
-h = "hours old"
+from sys import argv
 
-y = input("How old in years are you? ")
-t = input("How many hours has it been since midnight? ")
+script, y, m, t = argv
 
-print statement, y * days_in_a_year * hours_in_a_day + days_of_this_year * hours_in_a_day + t, h
\ No newline at end of file
+#statement = "You are" #start of the output
+hours_in_a_day = 24 #hours in one Earth day
+hours_in_year = 365 * hours_in_a_day #hours in one Earth year
+#h = "hours old" #statement that will be used to end the output
+
+#y = input("How old in years are you? ") #prompts the user to input their age in years.
+#m = input("How many days has it been since your last birthday? ") #237 #prompts the user to input the number of days since their last birthday
+#t = input("How many hours has it been since midnight? ") #prompts the user to input how many hours it has been since midnight
+
+y = int(y)
+m = int(m)
+t = int(t)
+
+def calc_hours_old(y, m, t):
```

Comment:
Hey Mr. Ortman,
I am having problems with assignment06. Could I get your help with it on Tuesday?